lib/components/wallet_connect/approve_token_transfer.dart

```diff
@@ -128,13 +147,16 @@ class _ApproveTokenTransferState extends State<ApproveTokenTransfer> {
                         }
                       }
                     } catch (e) {
+                      appLogger.d(e.toString());
                       setState(() {
                         isLoading = false;
                       });
 
                       if (mounted) {
                         Navigator.of(context).pop(ApproveTokenTransferResult(
-                            errorMessage: e.toString(), tick: null));
+                            errorMessage: e.toString(),
+                            tick: null,
```

Comment:
On error there is no need to send tick and transactionId as null.

Reply:
Fixed with 3f37417